### PR TITLE
Dalton - Edit Feature

### PR DESCRIPTION
**Pull request by DaltonJayPFG**
Thursday Session

### CODE REVIEW

File: dalton.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Line 2
```
Comment (PooleSamanthaPFG) [inline dalton.txt:2]: Add a Line 1

**Review by Jason-Wilson-PFG — COMMENTED** [review]
leaving a comment

File: dalton.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Line 3
```
Comment (SusmanNickiPFG) [inline dalton.txt:3]: Why is line 3 on line 2?

**Review by SusmanNickiPFG — CHANGES_REQUESTED** [review]
Only one change req

**Review by PooleSamanthaPFG — CHANGES_REQUESTED** [review]

File: dalton.txt (hunk 2 above)
Comment (DaltonJayPFG) [reply dalton.txt:3]: Just because.

Comment (SusmanNickiPFG) [reply dalton.txt:3]: Thanks for the fix

**Review by SusmanNickiPFG — APPROVED** [review]
Great job

**Review by DavisLeonPFG — APPROVED** [review]
Good Job

File: dalton.txt (hunk 1 above)
Comment (DaltonJayPFG) [reply dalton.txt:2]: Wake up.  I did change it.

Comment (PooleSamanthaPFG) [reply dalton.txt:2]: cry cry cry

**Review by PooleSamanthaPFG — APPROVED** [review]

**Pull request merged**